environment/deployments/science-platform/cloudsql/variables.tf

```diff
@@ -177,6 +186,7 @@ variable "butler_registry_dp02_ipv4_enabled" {
 variable "butler_registry_dp02_ssl_mode" {
   description = "Specify how SSL connection should be enforced in DB connections.  Options are ALLOW_UNENCRYPTED_AND_ENCRYPTED, ENCRYPTED_ONLY, and TRUSTED_CLIENT_CERTIFICATE_REQUIRED"
   type        = string
+  default     = "ALLOW_UNENCRYPTED_AND_ENCRYPTED"
```

Comment:
I mean it probably doesn't matter, but is there a good reason we're not enforcing encrypted transport?

Reply:
I agree encrypted transport should be set.  The setting is based on current state and that is the current setting for butler registry.